zerver/lib/import_realm.py

```diff
@@ -657,7 +657,7 @@ def import_uploads(realm: Realm, import_dir: Path, processes: int, processing_av
         else:
             bucket_name = settings.S3_AUTH_UPLOADS_BUCKET
         session = boto3.Session(settings.S3_KEY, settings.S3_SECRET_KEY)
-        bucket = session.resource('s3').Bucket(bucket_name)
+        bucket = session.resource('s3', endpoint_url=settings.S3_ENDPOINT_URL).Bucket(bucket_name)
```

Comment:
Should we just refactor this to call `get_bucket`?

Reply:
Maybe even better would be to refactor `get_bucket` to have the `session` argument be the second argument, and be optional, so that this code can not interact with `upload.py` details at all?